node.js/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,71 @@
+const { query } = require("express");
+const express = require("express");
+const app = express();
+let  reducer = (accumulator, currentValue) => accumulator + currentValue;
```

Comment:
You can also define reducer for all operations 

```suggestion
const add = (accumulator, currentValue) => Number(accumulator) + Number(currentValue);
const substract = (accumulator, currentValue) => Number(accumulator) - Number(currentValue);
const multiply = (accumulator, currentValue) => Number(accumulator) * Number(currentValue);
const divide = (accumulator, currentValue) => Number(accumulator) / Number(currentValue);

```